Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor

```diff
@@ -1,6 +1,96 @@
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.TabHeader Id="example-tabs">
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.TabHeaderTitle Active="true" Id="panel1">
+                Tab 1
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.TabHeaderTitle>
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.TabHeaderTitle Id="panel2">
+                Tab 2
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.TabHeaderTitle>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.TabHeader>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.TabContentWrapper LinkId="example-tabs">
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.TabContent Active="true" Id="panel1">
+                <p>Vivamus hendrerit arcu sed erat molestie vehicula. Sed auctor neque eu tellus rhoncus ut eleifend nibh porttitor. Ut in nulla enim. Phasellus molestie magna non est bibendum non venenatis nisl tempor. Suspendisse dictum feugiat nisl ut dapibus.</p>
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.TabContent>
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.TabContent Id="panel2">
+                <p>Suspendisse dictum feugiat nisl ut dapibus.  Vivamus hendrerit arcu sed erat molestie vehicula. Ut in nulla enim. Phasellus molestie magna non est bibendum non venenatis nisl tempor.  Sed auctor neque eu tellus rhoncus ut eleifend nibh porttitor.</p>
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.TabContent>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.TabContentWrapper>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Vertical Tabs</h2>
+        <p>
+            Add the attribute <Code>Vertical=true</Code> to the <Code Content="PFRhYkhlYWRlcj4="></Code> and <Code>Class="vertical"</Code> to the <Code Content="PFRhYkNvbnRlbnRXcmFwcGVyPg=="></Code> to stack tabs vertically.
```

Comment:
Add the attribute Vertical=true to the.... same thing
and Class="vertical" to the....

Reply:
They’re in the Code Content

Reply:
The English did not make sense